### PR TITLE
Fix ug bugs

### PR DESCRIPTION
**Pull request by KennyLewi**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -246,13 +254,17 @@ Adds a meet-up to the meet-up list.
...
 Format: `add s/MEETUP_SUBJECT i/MEETUP_INFO f/MEETUP_FROM t/MEETUP_TO [n/BUYER_NAME]…​`
 
-* New meet-ups must have at least one unique non-duplicate aspect from these three fields: MEETUP_SUBJECT, MEETUP_FROM, MEETUP_TO.
+* New meet-ups must have at least one unique non-duplicate aspect from these three fields: MEETUP_SUBJECT, MEETUP_FROM, MEETUP_TO. Else, it will be marked as a duplicate meet-up.
 
 * MEETUP_FROM and MEETUP_TO fields should follow the format `YYYY-MM-DD HH:MM`.
 
 * MEETUP_TO must be a date/time that is later than MEETUP_FROM.
 
-* Buyers that exist in buyer list will be marked as green, while those that don't will be marked as red.
```
Comment (NgZiXin) [inline docs/UserGuide.md:255]: Put in known issues also.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -232,6 +236,10 @@ Format: `clear`
 
 * Note how the mode is highlighted by `Viewing: Meet Ups` above the command line
 
+* Note how each meet-up contains buyer, if the buyer exists in the BuyerList, it will be marked as purple, else red. The matching here is done by case-sensitive matching, e.g. `Alex Yeoh` will only be purple if there is `Alex Yeoh` in the Buyer List, not `Alex yeoh` or `alex yeoh` or `Alex`.
```
Comment (NgZiXin) [inline docs/UserGuide.md:239]: dont use abbreviation like "won't" -> will not

File: docs/UserGuide.md [hunk 3]
```diff
@@ -351,15 +366,15 @@ Edits an existing property in the property list.
...
+* Edits the landlord at the specified `INDEX`. 
 * The index refers to the index number shown in the **displayed property list**. 
 * The index must be a positive integer 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
 
 Examples:
-*  `edit 1 p/91234567 s/100000` Edits the seller's phone number and the property's asking price to be `91234567` and `100,000` respectively.
-*  `edit 2 n/Betsy Crower` Edits the name of the 2nd property's seller to be `Betsy Crower`.
+*  `edit 1 p/91234567 s/100000` Edits the landlord's phone number and the property's asking price to be `91234567` and `100,000` respectively.
```
Comment (NgZiXin) [inline docs/UserGuide.md:376]: line 361 want to mention 1st property on the list?

**Comment by CreditableX** [comment]
LGTM

**Pull request merged**